common.eai

```diff
@@ -7425,10 +7410,7 @@ function TQHandleOnce takes nothing returns boolean
     if jobtimings - TimerGetElapsed(tq_timer) >= 1 then
       call Trace("WARN: Job " + Int2Str(jid) + " took a long time " + Real2Str(jobtimings - TimerGetElapsed(tq_timer)) + "s")
     endif
-    set unit_par = null
-    set group_par = null
-    set unit_par2 = null
-    return true
```

Comment:
Again return is more efficient than evaluating entire function.

Reply:
I haven't considered code efficiency here. What I'm considering is to minimize a few lines of code, reduce some strings, and decrease file size as much as possible